python/gps/algorithm/traj_opt/traj_opt_utils.py

```diff
@@ -85,7 +85,7 @@ def traj_distr_kl(new_mu, new_sigma, new_traj_distr, prev_traj_distr):
         )
 
     # Add up divergences across time to get total divergence.
-    return np.sum(kl_div)
+    return kl_div
```

Comment:
It seems like this change would break something in traj_opt_lqr.py... Was this a bug in the code?